README.md

```diff
@@ -173,6 +173,33 @@ const styles = StyleSheet.create({
 
 Aphrodite will ensure that the global `@font-face` rule for this font is only inserted once, no matter how many times it's referenced.
 
+### Caveats
+
+#### Assigning a string to a content property for a pseudo-element
+
+When assigning a string to the `content` property it requires double quotes in CSS.
```

Comment:
This is not necessarily true. Because this would work also:

```javascript
const styles = StyleSheet.create({
  large: {
      ':after': {
        content: "'Aphrodite'",
      },
    },
});
```

As would: 


```javascript
const styles = StyleSheet.create({
  large: {
      ':after': {
        content: `'Aphrodite'`,
      },
    },
});
```

What matters is that the value is a string which evaluates to a quoted string to be assigned to the `content` css property. Not sure of the best way to word that though...